src/applications/disability-benefits/all-claims/pages/unemployabilityDates.js

```diff
@@ -1,12 +1,12 @@
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
 
+import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
```

Comment:
whats this?

Reply:
This looks to have been apart of a resolving a conflict boondoggle. That line was moved up from where it was originally. Functionally not something I needed to touch for this ticket